lamindb_setup/core/_settings_store.py

```diff
@@ -49,6 +49,7 @@ def system_storage_settings_file():
 
 
 class InstanceSettingsStore(BaseSettings):
+    api_url: Optional[str] = None
```

Comment:
We also need the `schema_id`, here, don't we?

For `laminr` ([WP](https://www.notion.so/laminlabs/Build-an-R-SDK-laminR-8efbfd00157b4efe91b0927730492c0d)).

Reply:
Added.